packages/app/src/components/VisualizationOptions/VisualizationOptions.js

```diff
@@ -48,9 +48,7 @@ export class VisualizationOptions extends Component {
                     {content}
                     {helpText ? (
                         <Legend>
-                            <Help className={tabSectionOptionText.className}>
-                                {helpText}
-                            </Help>
+                            <Help>{helpText}</Help>
```

Comment:
Shouldn't `{tabSectionOptionText.styles}` https://github.com/dhis2/data-visualizer-app/pull/1265/files#diff-3db980ee16ade99ecb3d3a7c6c9cbfdbR110 be removed as well, as this class isn't used anymore?

Reply:
Fixed.